x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/ml/ml.test.ts

```diff
@@ -127,24 +127,6 @@ describe('ml_executor', () => {
     expect(result.warningMessages.length).toEqual(1);
   });
 
-  it('should report job missing errors as user errors', async () => {
```

Comment:
Can we modify this test to document how the ML executor handles job missing errors, now? I know we have the general test in `check_error_details.test.ts`, but another test at this level is relatively inexpensive and worthwhile, IMO.